modules/sdk-coin-zketh/src/zketh.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @prettier
+ */
+import { BaseCoin, BitGoBase, MPCAlgorithm } from '@bitgo/sdk-core';
+import { BaseCoin as StaticsBaseCoin, coins } from '@bitgo/statics';
+import { AbstractEthLikeNewCoins, TransactionBuilder as EthLikeTransactionBuilder } from '@bitgo/abstract-eth';
+import { TransactionBuilder } from './lib';
+
+export class Zketh extends AbstractEthLikeNewCoins {
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Zketh(bitgo, staticsCoin);
+  }
+
+  protected getTransactionBuilder(): EthLikeTransactionBuilder {
+    return new TransactionBuilder(coins.get(this.getBaseChain()));
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return false;
+  }
+
+  /** @inheritDoc */
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'ecdsa';
+  }
+}
```

Comment:
There is no need for supportsTss and getMPCAlgorithm as we don't have tss support as of now. Please remove them in the follow up Pr that you are working on

Reply:
Removed in the next PR.